app/auth/templates/auth/login.html

```diff
@@ -83,12 +83,11 @@ <h1 class="h2 text-center">Welcome back!</h1>
             <i class="fa fa-facebook"></i> Sign in with Facebook
           </a>
         </div>
-        <div class="text-center p-3" style="font-size: 12px; font-weight: light; margin: auto">
+        <div class="text-center p-3" style="font-size: 12px; font-weight: lighter; margin: auto">
```

Comment:
Just tried locally and using `lighter` is maybe too light :) and makes the text hard to read.